src/store/safePass.ts

```diff
@@ -0,0 +1,36 @@
+import { cgwDebugStorage } from '@/components/sidebar/DebugToggle'
+import { IS_PRODUCTION, GATEWAY_URL_PRODUCTION, GATEWAY_URL_STAGING } from '@/config/constants'
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+
+const GATEWAY_URL = IS_PRODUCTION || cgwDebugStorage.get() ? GATEWAY_URL_PRODUCTION : GATEWAY_URL_STAGING
+const GLOBAL_CAMPAIGN_IDS: Record<'1' | '11155111', string> = {
+  '11155111': 'fa9f462b-8e8c-4122-aa41-2464e919b721',
+  '1': '9ed78b8b-178d-4e25-9ef2-1517865991ee',
+}
```

Comment:
Do you know if its possible to fetch points without specifying a campaign id? i.e. the CGW returning points by campaign id for a given address. That way we wouldn't need to hard-code this.

Reply:
Its not possible. There is no global endpoint for points.